### ISSUE TITLE
"Tab" key now switches from Character to Flea Market only

### ISSUE THREAD

**Issue by lexiconvict** [tyfon7/UIFixes#59]
After recently updating to the new version [Version 2.4.3], it seems like the "Tab" key function has changed. I believe it previously would switch between the Character tab to the last tab you were on, but now it only goes between Character and Flea Market tabs no matter what. I used it frequently to quick switch between Character and Traders, and now cannot do that. I'm not sure if this is intended behaviour or perhaps something wrong on my end, but I wanted to bring it to your attention. I'd be more than happy to provide you with more information and details to help if you need it, just let me know.

Cheers, and thank you so much for your time and effort in creating an incredibly helpful QOL mod that I adore.

-Josh

**Comment by tyfon7** [comment]
I can't repro this - I don't touch this UI interaction. It doesn't make a lot of sense, as tab is "switch" functionality, it just opens and closes the character panel. There's no logic to specifically go to the flea market, it just closes the character panel and you should be back wherever you were.

It's possible that this is either a different mod you have, or a strange interaction/conflict with another mod. Can you confirm this behavior is fixed when you uninstall UI Fixes?

**Comment by lexiconvict** [comment]
Hi Tyfon, I also installed your auto run mod, some trader mods, as well as the "QuickSell" mod by Tadikas that same day you released the update for "UI Fixes". After going back through and isolating each one, the problem was resolved when removing the "QuickSell" mod. I apologize for making the assumption it was your mod, I don't know why I didn't question whether it was any of the other ones. In my head, I jumped right to the conclusion that it must've been a change with your update for some dumbe reason. I'm very sorry if I wasted any of your time, I'll be more careful in the future with bug reports! Cheers.
